DeveloperGuide.html

```diff
@@ -0,0 +1,567 @@
+<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]--><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta name="generator" content="Asciidoctor 1.5.6.1"><title>FoodieBot - Developer Guide</title><link rel="stylesheet" href="stylesheets/gh-pages.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.5.0/css/font-awesome.min.css">
```

Comment:
> As a more OOP model, we can store a Tag list in FoodieBot, which Canteen can reference. This would allow FoodieBot to only require one Tag object per unique Tag, instead of each Canteen needing their own Tag object.

This sentence doesn't seem to match with the associated diagram